<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReleaseMission</class>
 <widget class="QMainWindow" name="ReleaseMission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>471</width>
      <height>320</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>任务名</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="e_missionName">
         <property name="font">
          <font>
           <family>Mongolian Baiti</family>
           <pointsize>13</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>任务简介</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="e_missionIntro"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>任务文本</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="e_srcText"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>源语言</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Chn_1">
         <property name="text">
          <string>汉语</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Eng_1">
         <property name="text">
          <string>英语</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Fre_1">
         <property name="text">
          <string>法语</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Spn_1">
         <property name="text">
          <string>西班牙语</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>目标语言</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Chn_2">
         <property name="text">
          <string>汉语</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Eng_2">
         <property name="text">
          <string>英语</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Fre_2">
         <property name="text">
          <string>法语</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Spn_2">
         <property name="text">
          <string>西班牙语</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="continue_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>继续</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发布任务</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
